src/realm/util/interprocess_mutex.hpp

```diff
@@ -112,9 +122,11 @@ class InterprocessMutex {
     friend class InterprocessCondVar;
 };
 
-
 inline InterprocessMutex::InterprocessMutex()
 {
+#ifdef REALM_ROBUST_MUTEX_EMULATION
+    std::call_once(static_flag, static_initialisation);
```

Comment:
Simon says: This could (from c++11) be replaced with
~~~
{ static int dummy = static_initialization(); }
~~~
provided static_initialization is made to return an int to fit. So call_once is not needed.